AmericanMedicalAssociation/AMA-style-guide
# AMA-104: adds layout molecule templates.

https://palantir.atlassian.net/browse/AMA-104

Adds the One Up Layout molecule, Two Up Layout molecule, Two Up Heavy Left Layout molecule, and Three Up Layout molecule (desktop only) using sass-grid flexbox.

### Steps to test:

1. Navigate to Molecules > Layouts > One Up Layout
2. Observe that the One Up Layout molecule is present and works as expected.
3. Navigate to Molecules > Layouts >Two Up Layout
4. Observe that the Two Up Layout molecule is present and works as expected.
5. Navigate to Molecules > Layouts >Two Up Heavy Left Layout
6. Observe that the Two Up Heavy Left Layout molecule is present and works as expected.
7. Navigate to Molecules > Layouts >Three Up Layout
8. Observe that the Three Up Layout molecule is present and works as expected.
